Packs/MicrosoftExchangeOnline/Playbooks/playbook-O365-SecurityAndCompliance-Search.yml

```diff
@@ -982,6 +982,8 @@ tests:
 - O365-SecurityAndCompliance-Test
 - O365-SecurityAndCompliance-ContextResults-Test
 supportedModules:
+- C3
+- X0
 - X1
```

Comment:
SecurityAndCompliance (v1) and "EWS Extension Online Powershell v2" can be removed from all new modules c1,c3,x0